parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -31,6 +32,10 @@ COREOS_OS_NAME="COREOS"
 # Set default kubectl
 KUBECTL=/usr/local/bin/kubectl
 
+MASTER_OFFSET=$(echo `hostname` | cut -d'-' -f 4)
```

Comment:
If anyone knows better way to get the master number from the custom script please let me know... I tried passing in copyIndex(masterOffset) in provisionScriptParametersMaster but I can't use copyIndex() in that context

Reply:
take a look at configure-swarm-cluster.sh where ```MASTERCOUNT=${3}```.  This can be set on the incoming env variables.